lib/newsUtils.js

```diff
@@ -0,0 +1,27 @@
+import BASE from './common';
+
+function createNewsItemData(newsItem) {
+  return {
+    date: new Date(newsItem.get('Created')),
+    description: newsItem.get('Description'),
+    title: newsItem.get('Title'),
+    id: newsItem.get('ID')
+  };
+}
+
+const getNewsItems = function async() {
```

Comment:
Maybe add a TODO if you want to ideally replace this to use something from a future version of the schema generator? Just so it's easy to go back to this

Reply:
Addressing on the new branch (`annie/airtable`)